content/docs/concepts/network/index.md

```diff
@@ -0,0 +1,437 @@
+---
+title: The Istio network configuration objects and your application
```

Comment:
This is a bit long for our normal document titles, can this be made shorter?

Reply:
done